src/screen.cpp

```diff
@@ -443,10 +443,10 @@ void Screen::drawWidgets() {
                 h = (bounds[2] - bounds[0]) / 2;
             }
             nvgGlobalAlpha(mNVGContext,
-                           std::min(1.0, 2 * (elapsed - 0.5f)) * 0.8);
+                           std::min(1.05, 2 * (elapsed - 0.5f)) * 0.8);
```

Comment:
Yes, you could add a theme variable for this. (mTooltipOpacity?)

Reply:
That can be done!  The only snag is that the side-by-side stuff in the repo with theme builder got pretty obtuse, `make_compare_window`(https://github.com/svenevs/nanogui-customization-demo/blob/77a95c38501f9e50f0ac7770e4991df632130bff/cpp/make_compare_window.hpp#L50) is beastly.

1. Which widgets would you like to show up in the side-by-side comparison window?

    The linked function has all of that because each of those is needed for testing the theme changes.  But having all of that code in the example would distract from the purpose.

2. I switched from the light theme to a not-so-great "solarized light"-esque theme in that repo.  You want this example to have a more traditional light theme (like what's in the pictures above), right?

3. Can I include the `json` comment with a link to the `theme-builder`?